Sources/AuthFoundation/JWT/Extensions/ClaimConvertable+Extensions.swift

```diff
@@ -12,25 +12,42 @@
 
 import Foundation
 
+@_documentation(visibility: private)
```

Comment:
curious, so this hides the extension from generated docs or from Xcode as well?

Reply:
From the generated docs. Many of these just clutter up the docs. For reference, here's [the list of underscored attributes](https://github.com/swiftlang/swift/blob/main/docs/ReferenceGuides/UnderscoredAttributes.md).